### DISCUSSION TITLE
How to add Predefined Metadata Definitions

### DISCUSSION THREAD

**Discussion by sumant-agnihotri** [pimcore/pimcore#11778] [Q&A / Support] [answered]
Hi all, I set up Pimcore community edition on my local server. Now I want to add some Metadata definitions (in settings). But the UI add button is disabled and I cant find any documentation on how to do it from the backend/database. Can some please point me the right direction.
Thanks!

**Comment by dvesh3** [comment, marked as answer]
The config button is disabled if the storage write target is set to `symfony-config` or not set at all and debug mode is disabled. please read more about config storage locations here https://pimcore.com/docs/pimcore/current/Development_Documentation/Deployment/Configuration_Environments.html#page_Configuration-Storage-Locations-Fallbacks

Might be interesting for you https://github.com/pimcore/skeleton/pull/86

**Reply by sumant-agnihotri** [reply]
Thanks this helps a lot!
